spec/predicates/baseline.md

```diff
@@ -0,0 +1,367 @@
+  "predicateType": "https://baseline.openssf.org/attestation/0.1",
+  "predicate": {
+    "author": { "uri": "https://github.com/evankanderson" },
+    "framework": "https://baseline.openssf.org/versions/2025-10-10",
+    "assessedAt": "2025-11-20T23:15:00Z",
+    "controls": [
+      {
+        "control": "OSPS-BR-03.01",
+        "result": "passed",
+        "evidence": [
+          {
+            "name": "https://github.com/in-toto/attestation",
+            "result": "passed",
+            "description": "HTTPS URL"
+          },
+          {
+            "name": "https://in-toto.io/",
+            "result": "passed",
+            "description": "HTTPS URL"
+          }
+        ]
+      },
+      {
+        "control": "OSPS-BR-03.02",
+        "result": "passed"
+        // evidence is optional, not required.
+      }
+    ]
+  }
+}
+```
+
+This predicate has been somewhat adopted from the [Gemara layer 4] format.
+
+### Parsing Rules
+
+The baseline predicate follows the [in-toto parsing rules]. In particular:
+
+- The baseline admits the possibility of extension fields as a means of
+  developing and evolving the base specification. Specific software
+  implementations MAY emit additional control assessment and evidence data as
+  long as that data follows the monotonic principle: that is, if the field were
+  omitted, it would not transform a DENY decision into an ALLOW.
+- Consumers MUST accept and ignore unrecognized fields in the predicate.
+- While the version of the baseline predicate is "0.1" to indicate that feedback
+  and improvement is desired, it is expected that efforts will be made to ensure
+  backwards compatibility between "0.1" and future versions.
+
+The following fields are considered mandatory:
+
+- `author` and `framework` in ProjectAssessment.
+- `control` and `result` in ControlAssessment.
+
+Additionally, the `result` field of a control assessment in a baseline predicate
+is bounded by (no stronger than) the `result`s of the corresponding assessment
+results (`evidence`):
+
+- If there is at least one "failure" in the evidence results, the control's
+  result is "failure".
+- If there are no "failure" results but at least one "needs review" evidence
+  result, the control's result may only be "needs review" or "failure".
+- A "passed" result for the control requires that all evidence either has a
+  "passed" result or does not include an explicit result.
+
+Predicates which do not properly aggregate the evidence results into control
+results MAY be discarded by consumers.
+
+### Fields
+
+`predicateType` _string ([TypeURI]), required_
+
+> Identifier for the schema of the baseline predicate. Always
+> `https://baseline.openssf.org/attestation/0.1` for this version of the spec.
+
+`predicate.author` _object ([ResourceDescriptor]), required_
+
+> Identifier for the entity (either a human or software program) which assessed
+> the control implementations for the project.
+
+`predicate.framework` _string, required_
```

Comment:
aside: Do you foresee this applying to other control frameworks as well? i.e., outside of baseline